### DISCUSSION TITLE
Should multiple zones shown as multiple devices?

### DISCUSSION THREAD

**Discussion by Andre0512** [Andre0512/hon#45] [Ideas]
For some appliances (REF, IH and maybe OV) the api exposes multiple zones. For a hob each plate is a zone, for a fridge exists the refrigerator, the freezer and a custom zone and an oven seems to could have also different unites. 
A zone has always the same attributes: temperature, power, pan status, door open, start program, on/off and so on, e.g have a look at the hob attributes in #22. I think it's useful to have a main device for common things  in home assistant and for each plate an own device, so I implemented the logic to always split up zones.
But now I try to add fridges in #41 but here I'm not sure if we should really always split up zones or make an exception and show all "fridge zones" as one device. Hobs can have 3 or 4 or even 6 plates, but fridges have a freezer or not have a freeze so that remains manageable I think.
What are your suggestions? Should we go different ways for hobs and fridges here? Or do you always like to have one or multiple devices for zones in home assistant?

**Comment by sonyboypatrick** [comment]
Hello André, I think it would be nice to have a fridge/freezer combination as a single devive with multiple 
attributes. Creating one devive per zone would blow up everything for such a little thing.
